otter/test/worker/test_launch_server_v1.py

```diff
@@ -2224,6 +2220,10 @@ def setUp(self):
             self, 'otter.worker.launch_server_v1.remove_from_load_balancer')
         self.remove_from_load_balancer.return_value = succeed(None)
 
+        # reset it to empty in beginning of test as it is module-wise
+        # info retained in every test
+        launch_server_v1._semaphores = {}
```

Comment:
Not only should this use `patch` (as above) but I feel like it should be in a method or something; repeating both the same code and the same comment is a code smell.